marrow/mailer/transport/ses.py

```diff
@@ -1,8 +1,8 @@
 # encoding: utf-8
 
 try:
-    import boto.ses 
-    from boto.ses import SESConnection
+    import boto3
+    from botocore.exceptions import ClientError
 
 except ImportError:
     raise ImportError("You must install the boto package to deliver mail via Amazon SES.")
```

Comment:
This should be updated to reference the correct package.

Reply:
> This is looking great. Minor formatting corrections requested, and a question about the possibility of missing response keys.

Thanks for looking into this. 
1. I adjusted the flow in `send_raw_email`
2. Autoformat using `autopep8`
3. Not sure about above suggestion of `updated to reference the correct package.` I though I already commit the change to boto3.

The only thing which I was thinking of, this can be a breaking change for persons who're using boto. But since AWS stops support for signature version 3 you should upgrade to signature 4 which one works with boto3. 
How do you thing this must be handled? Major release to indicates it's a breaking change?

Reply:
Again, that's very specific when I was referring to `…install the boto package…` within the exception mentioning `boto3` instead. PEP-8 is [unfortunate](https://medium.com/@amcgregor/your-code-style-guide-is-crap-but-still-better-than-nothing-c4d75f5b3c44). 😕